include/variables.h

```diff
@@ -50,4 +50,6 @@ extern u16 D_800A0940;
 extern u16 D_800A0942;
 extern s16 D_800A0944;
 
+extern f32 D_BF800000;
```

Comment:
as mentioned on discord, 0xBF800000 isn't an address, it's just the raw hex representation of -1.0f